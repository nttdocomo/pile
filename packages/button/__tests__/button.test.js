/**
 * @author renmaomin@126.com
 */
import React from 'react';
import { shallow } from 'enzyme';
import Button from '../src';

describe('Button suite', () => {
  it('render a button', () => {
    const wrapper = shallow(<Button>default</Button>);
    expect(wrapper).toMatchSnapshot();
  });
});
