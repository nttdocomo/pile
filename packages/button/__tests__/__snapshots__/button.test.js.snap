// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button suite render a button 1`] = `""`;
